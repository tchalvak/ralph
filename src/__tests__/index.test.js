import React from "react"
import renderer from "react-test-renderer"
import IndexPage from "../pages/index"

describe("IndexPage", () => {
  it("renders correctly", () => {
    const location = {
      pathname: "/",
    }

    const tree = renderer.create(<IndexPage location={location} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})