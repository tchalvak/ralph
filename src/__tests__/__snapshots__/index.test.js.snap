// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<div>
  <div
    className="body is-loading "
  >
    <div
      id="wrapper"
    >
      <header
        id="header"
        style={Object {}}
      >
        <div
          className="logo"
        >
          <span
            className="icon fa-diamond"
          />
        </div>
        <div
          className="content"
        >
          <div
            className="inner"
          >
            <h1>
              Ralph
            </h1>
            <p>
              An AI integration showcase.
            </p>
          </div>
        </div>
        <nav>
          <ul>
            <li>
              <a
                href="javascript:;"
                onClick={[Function]}
              >
                Intro
              </a>
            </li>
            <li>
              <a
                href="javascript:;"
                onClick={[Function]}
              >
                Work
              </a>
            </li>
            <li>
              <a
                href="javascript:;"
                onClick={[Function]}
              >
                About
              </a>
            </li>
            <li>
              <a
                href="javascript:;"
                onClick={[Function]}
              >
                Contact
              </a>
            </li>
          </ul>
        </nav>
      </header>
      <div
        id="main"
        style={
          Object {
            "display": "none",
          }
        }
      >
        <article
          className=" "
          id="intro"
          style={
            Object {
              "display": "none",
            }
          }
        >
          <h2
            className="major"
          >
            Understanding 
            <br />
            Artificial Intelligence
          </h2>
          <span
            className="image main"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </span>
          <p>
            Artificial Intelligence today is a wholly different landscape from the science fiction ideas that were written about 50 years ago. For example, we have a variety of speech interfaces in the form of devices and tools like Alexa, Siri, Google Home, and more, devices in a box that are nothing similar to the idea of a human in a box.
          </p>
          <p>
            We give these things a voice because they are a new way of working with your computer. One part of the AI movement is an introduction of new tools; speech instead of a keyboard, speech instead of a screen.
          </p>
          <p>
            Another practical aspect of the AI movement is a force multiplication of simple decision-making. Customer support is an example of this.
          </p>
          <p>
            I can put a customer service support chat on my website so that users can talk to me directly, but if they are in a different timezone, I'm unavailable. By using an AI, or chatbot, I can provide a reduced experience, for more time in the day and night. It would always be better if they could talk to me, I'm really good at talking and helping, but if the choice is between me for 1 hour a day and a much less effective chat-bot for 24 hours a day, I am going to give that up to the AI.
          </p>
          <div
            className="close"
            onClick={[Function]}
          />
        </article>
        <article
          className=" "
          id="work"
          style={
            Object {
              "display": "none",
            }
          }
        >
          <h2
            className="major"
          >
            Work
          </h2>
          <span
            className="image main"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </span>
          <p>
            Adipiscing magna sed dolor elit. Praesent eleifend dignissim arcu, at eleifend sapien imperdiet ac. Aliquam erat volutpat. Praesent urna nisi, fringila lorem et vehicula lacinia quam. Integer sollicitudin mauris nec lorem luctus ultrices.
          </p>
          <p>
            Nullam et orci eu lorem consequat tincidunt vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus pharetra. Pellentesque condimentum sem. In efficitur ligula tate urna. Maecenas laoreet massa vel lacinia pellentesque lorem ipsum dolor. Nullam et orci eu lorem consequat tincidunt. Vivamus et sagittis libero. Mauris aliquet magna magna sed nunc rhoncus amet feugiat tempus.
          </p>
          <div
            className="close"
            onClick={[Function]}
          />
        </article>
        <article
          className=" "
          id="about"
          style={
            Object {
              "display": "none",
            }
          }
        >
          <h2
            className="major"
          >
            About
          </h2>
          <span
            className="image main"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </span>
          <p>
            Lorem ipsum dolor sit amet, consectetur et adipiscing elit. Praesent eleifend dignissim arcu, at eleifend sapien imperdiet ac. Aliquam erat volutpat. Praesent urna nisi, fringila lorem et vehicula lacinia quam. Integer sollicitudin mauris nec lorem luctus ultrices. Aliquam libero et malesuada fames ac ante ipsum primis in faucibus. Cras viverra ligula sit amet ex mollis mattis lorem ipsum dolor sit amet.
          </p>
          <div
            className="close"
            onClick={[Function]}
          />
        </article>
        <article
          className=" "
          id="contact"
          style={
            Object {
              "display": "none",
            }
          }
        >
          <h2
            className="major"
          >
            Contact
          </h2>
          <form
            action="#"
            method="post"
          >
            <div
              className="field half first"
            >
              <label
                htmlFor="name"
              >
                Name
              </label>
              <input
                id="name"
                name="name"
                type="text"
              />
            </div>
            <div
              className="field half"
            >
              <label
                htmlFor="email"
              >
                Email
              </label>
              <input
                id="email"
                name="email"
                type="text"
              />
            </div>
            <div
              className="field"
            >
              <label
                htmlFor="message"
              >
                Message
              </label>
              <textarea
                id="message"
                name="message"
                rows="4"
              />
            </div>
            <ul
              className="actions"
            >
              <li>
                <input
                  className="special"
                  type="submit"
                  value="Send Message"
                />
              </li>
              <li>
                <input
                  type="reset"
                  value="Reset"
                />
              </li>
            </ul>
          </form>
          <ul
            className="icons"
          >
            <li>
              <a
                className="icon fa-twitter"
                href="#"
              >
                <span
                  className="label"
                >
                  Twitter
                </span>
              </a>
            </li>
            <li>
              <a
                className="icon fa-facebook"
                href="#"
              >
                <span
                  className="label"
                >
                  Facebook
                </span>
              </a>
            </li>
            <li>
              <a
                className="icon fa-instagram"
                href="#"
              >
                <span
                  className="label"
                >
                  Instagram
                </span>
              </a>
            </li>
            <li>
              <a
                className="icon fa-github"
                href="#"
              >
                <span
                  className="label"
                >
                  GitHub
                </span>
              </a>
            </li>
          </ul>
          <div
            className="close"
            onClick={[Function]}
          />
        </article>
      </div>
      <footer
        id="footer"
        style={Object {}}
      >
        <p
          className="copyright"
        >
          Design by: 
          <a
            href="https://html5up.net"
          >
            HTML5 UP
          </a>
          .
        </p>
      </footer>
    </div>
    <div
      id="bg"
    />
  </div>
</div>
`;
